northstar/build.rs

```diff
@@ -12,12 +12,75 @@
 //   See the License for the specific language governing permissions and
 //   limitations under the License.
 
+use std::env;
 use vergen::{generate_cargo_keys, ConstantsFlags};
 
 fn main() {
+    println!("cargo:rerun-if-changed=build.rs");
+    println!("cargo:rerun-if-changed=hello_world/src/main.rs");
+    println!("cargo:rerun-if-changed=hello_world/Cargo.toml");
+    println!("cargo:rerun-if-changed=hello_world/manifest.yaml");
     let flags = ConstantsFlags::BUILD_TIMESTAMP
         | ConstantsFlags::TARGET_TRIPLE
         | ConstantsFlags::SHA_SHORT
         | ConstantsFlags::SEMVER_FROM_CARGO_PKG;
     generate_cargo_keys(flags).expect("Unable to generate the cargo keys!");
+
+    package_hello_example();
+}
+
+pub fn package_hello_example() {
+    use std::{fs, path::Path, process::Command};
+    use tempfile::tempdir;
+
+    let out_dir_var = env::var_os("OUT_DIR").unwrap();
+    let out_dir_path = Path::new(&out_dir_var);
+    let out_dir_string = out_dir_var.to_string_lossy();
+
+    let args = ["build", "--release", "--target-dir", &out_dir_string];
+    let mut cmd = Command::new("cargo");
```

Comment:
In the integration tests we use `escargot` to build the `test_container`. Could it be maybe useful here?

Reply:
never heard about that before ;) but using cargo here seems pretty straightforward to me